### PR TITLE
EICNET-777: Implement search suggestions (Did you mean?)

### PR DESCRIPTION
**Pull request by didierdemaeyer**
Installed the `search_api_spellcheck` module and added the "Did you mean" spellcheck in the header of the global and group search pages.

### Issues

The spellcheck block is displayed on the group search page (/{group}/search-group), but not on the global search page (/search).

### CODE REVIEW

**Review by 4hardfire — COMMENTED** [review]
@didierdemaeyer, I've added the missing header in the twig template used in the global view.

File: composer.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
         "drupal/scheduler": "^1.3",
         "drupal/scheduler_content_moderation_integration": "^1.3",
         "drupal/search_api_solr": "^4.1",
-        "drupal/search_api_sorts": "^1.0",
```
Comment (4hardfire) [inline composer.json:51]: @didierdemaeyer Did you remove the module **search_api_sorts** by mistake? I checked all the configurations and the module seems to be used.
Can you have a look?

**Review by 4hardfire — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (didierdemaeyer) [reply composer.json:51]: Yes, I must've removed that by accident. I've re-added it.

**Review by 4hardfire — APPROVED** [review]
Seems ok to me!

**Pull request merged**